org.eclipse.scanning.command/src/org/eclipse/scanning/command/factory/GridModelExpresser.java

```diff
@@ -40,6 +40,9 @@ public String pyExpress(GridModel model, Collection<IROI> rois, boolean verbose)
 			+(verbose
 				? (", snake="+(model.isSnake()?"True":"False"))
 				: (model.isSnake()?"":", snake=False"))
+			+(verbose
+				? (", continuous="+(model.isContinuous()?"True":"False"))
+				: (model.isContinuous()?"":", continuous=False"))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)